src/bson.c

```diff
@@ -62,6 +66,9 @@
 
 #define PHONGO_ODM_FIELD_NAME "__pclass"
 
+#define IS_CLASS_INSTANTIATABLE(ce) \
```

Comment:
It's common to prefix macros, or, if you don't, #undefine them when you're done with them